sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/dao/EvaluationDaoImpl.java

```diff
@@ -25,8 +25,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.apache.commons.lang.StringUtils;
```

Comment:
org.apache.commons.lang3.StringUtils ?

Reply:
Project uses the original so I didn't migrate it!